docs: add common issue for code node cant access credentials (#4300)
n8n-io/n8n-docs: docs/integrations/builtin/core-nodes/n8n-nodes-base.code/common-issues.md

/// info | Use Remove Duplicates when possible
If you're interested in using variables to avoid processing the same data items more than once, consider using the [Remove Duplicates node](/integrations/builtin/core-nodes/n8n-nodes-base.removeduplicates/index.md) instead. The Remove Duplicates node can save information across executions to avoid processing the same items multiple times.
///

> ## Cannot access credentials in a code node

Reviewer: 🚨 **ERROR: Use contractions** (Must fix)

Use "can't" instead of "cannot" for a conversational tone.

```suggestion
## Can't access credentials in a code node
```

> By design, Code nodes cannot access credentials. They don't have access to n8n’s internal credential management system. This prevents exposure of sensitive authentication data.

Reviewer: 🚨 **ERROR: Use contractions** (Must fix)

Use "can't" instead of "cannot" for a conversational tone.

```suggestion
By design, Code nodes can't access credentials. They don't have access to n8n’s internal credential management system. This prevents exposure of sensitive authentication data.
```

> Attempts to reference credentials in a Code node using expressions or methods like `this.getCredentials()` or `$getCredentials()` will result in errors, such as `this.getCredentials is not a function` and `$getCredentials is not defined`. 

Reviewer: 💡 **SUGGESTION: Simplify word**

Consider using "try" instead of "attempt" for clarity.

```suggestion
trys to reference credentials in a Code node using expressions or methods like `this.getCredentials()` or `$getCredentials()` will result in errors, such as `this.getCredentials is not a function` and `$getCredentials is not defined`. 
```

> Attempts to reference credentials in a Code node using expressions or methods like `this.getCredentials()` or `$getCredentials()` will result in errors, such as `this.getCredentials is not a function` and `$getCredentials is not defined`. 

Reviewer: 🚨 **ERROR: Use contractions** (Must fix)

Use "isn't" instead of "is not" for a conversational tone.

```suggestion
Attempts to reference credentials in a Code node using expressions or methods like `this.getCredentials()` or `$getCredentials()` will result in errors, such as `this.getCredentials isn't a function` and `$getCredentials isn't defined`. 
```
